### ISSUE TITLE
Nextcloud installation broken by the update to 1.3.0

### ISSUE THREAD

**Issue by Flaburgan** [nextcloud/mail#2911]
Yesterday, I clicked to update the nextcloud mail app from 1.1.4 to 1.3.0.
In the same time, I had the app open in another tab. Suddenly, both tabs stopped worked and displayed the "maintenant mode" page of nextcloud. It is stuck in that state since then, even after a server reboot.

I'm using the snap package for nextcloud, please tell me where I can find the logs.

### Expected behavior
Nextcloud works normally

### Actual behavior
Nextcloud is stuck on the "maintenance mode" page, even after server reboot

### Mail app

**Mail app version:** no idea if still 1.1 or 1.3, it was during the upgrade

**Mailserver or service:** IMAP provided by OVH

### Server configuration

**Operating system**: Ubuntu 18.04

**Web server:** snap package of nextcloud

**Nextcloud Version:** 17

**Comment by ChristophWurst** [comment]
The Nextcloud mail app offers an extensive logging system to make it easier identifying and tracking down bugs. Please enable [debug mode](https://docs.nextcloud.com/server/stable/developer_manual/general/devenv.html#enabling-debug-mode) and set the log [level to debug](https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/logging_configuration.html) in your admin settings. Then, try to reproduce your issue and take another look at `data/nextcloud.log`, `data/horde_imap.log` and `data/horde_smtp.log`.

**Comment by ChristophWurst** [comment]
So? Any insights?

**Comment by Flaburgan** [comment]
I'm sorry but as this is a snap installation and I am not used at snap at all, I'm not sure where to look. The only `nextcloud.log` file I found is in`/var/snap/nextcloud/common/nextcloud/data/nextcloud.log` and it hasn't been updated since yesterday morning. Restarting nextcloud with `snap restart nextcloud` doesn't change anything at that.

I tried to find the config file with 
```
locate config/config.php
/snap/nextcloud/19299/htdocs/config/config.php
/snap/nextcloud/20007/htdocs/config/config.php
/var/snap/nextcloud/19299/nextcloud/config/config.php
/var/snap/nextcloud/20007/nextcloud/config/config.php
```
The last answer seemed to be the more recent so I edited it setting the log level to 2 and restarting nextcloud but once again, nothing new in the log. In that config file btw the `maintenance` parameter is `false`, but I am still stuck with "nextcloud is in maintenance mode" in my browser. So I am not sure I am modifying the right file.

Pinging @kyrofa as other snap users could be affected.
